fixtures/american-dj/inno-spot-pro.json

```diff
@@ -0,0 +1,522 @@
+  "name": "Inno Spot Pro",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Ken Harris"],
+    "createDate": "2022-03-19",
+    "lastModifyDate": "2022-03-19"
+  },
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/6922/inno_spot_pro.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/inno-spot-pro"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=qZIVSOfQyBE"
+    ]
+  },
+  "physical": {
+    "dimensions": [288, 425, 167],
+    "weight": 8.5,
+    "power": 140,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "1× 80W White LED"
+    },
+    "lens": {
+      "degreesMinMax": [12, 17]
+    }
+  },
+  "wheels": {
+    "Color Wheel": {
+      "slots": [
+        {
+          "type": "Open"
+        },
+        {
+          "type": "Color",
+          "name": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "type": "Color",
+          "name": "Orange",
+          "colors": ["#ff7f00"]
+        },
+        {
+          "type": "Color",
+          "name": "Yellow",
+          "colors": ["#ffff00"]
+        },
+        {
+          "type": "Color",
+          "name": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "type": "Color",
+          "name": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "type": "Color",
+          "name": "Purple",
+          "colors": ["#b803b8"]
+        },
+        {
+          "type": "Color",
+          "name": "Light Blue",
+          "colors": ["#007fff"]
+        },
+        {
+          "type": "Color",
+          "name": "Pink",
+          "colors": ["#ff6aff"]
+        }
+      ]
+    },
+    "Gobo Wheel": {
+      "slots": [
+        {
+          "type": "Gobo",
+          "name": "Spot"
+        },
+        {
+          "type": "Gobo",
+          "name": "Gobo 1"
+        },
+        {
+          "type": "Gobo",
+          "name": "Gobo 2"
+        },
+        {
+          "type": "Gobo",
+          "name": "Gobo 3"
+        },
+        {
+          "type": "Gobo",
+          "name": "Gobo 4"
+        },
+        {
+          "type": "Gobo",
+          "name": "Gobo 5"
+        },
+        {
+          "type": "Gobo",
+          "name": "Gobo 6"
+        }
+      ]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "fineChannelAliases": ["Pan fine"],
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "540deg"
+      }
+    },
+    "Tilt": {
+      "fineChannelAliases": ["Tilt fine"],
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "0deg",
+        "angleEnd": "240deg"
+      }
+    },
+    "Color Wheel": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 7],
+          "type": "WheelSlot",
+          "slotNumber": 1,
+          "helpWanted": "The colors in the manual don't match the color wheel on the product page."
```

Comment:
The video (at 0:33-0:36) shows the fixture cycling through the color wheel, and it's [red, orange, yellow, green, blue, purple, light blue, pink, white] -- same as the user manual and the "cut sheet" -- so I think the [white, red, blue, green, ...] picture on the webpage is just wrong.